src/components/elements/off-canvas-menu.js

```diff
@@ -65,11 +65,11 @@ const OffCanvasMenuWrapper = props => {
             <OffCanvasMenuContainer>
                 <BackArrow onClick={handleArrowClick} />
                 <div>
-                    <StyledLink to="/keep-safe/" onClick={handleArrowClick}>
-                        Keep safe
+                    <StyledLink to="/about-us/" onClick={handleArrowClick}>
+                        {localize('About us')}
                     </StyledLink>
```

Comment:
If it's `Button Link`, we have added a `LinkButton` recently.  
`import { LinkButton } from 'components/form'`

Reply:
no its not button, its just a red link text 🙏